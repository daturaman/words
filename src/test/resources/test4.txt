%20 @ $40.00 &&&& %%%