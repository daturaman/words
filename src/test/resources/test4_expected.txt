Word count = 3
Average word length = 3.333
Number of words of length 1 is 1
Number of words of length 3 is 1
Number of words of length 6 is 1
The most frequently occurring word length is 1, for word lengths of 1 & 3 & 6